README.md

```diff
@@ -1,16 +1,19 @@
-# Gravitational Teleport
+# Teleport
 
-Gravitational Teleport is a modern security gateway for remotely accessing:
-* Clusters of Linux servers via SSH or SSH-over-HTTPS in a browser.
-* Kubernetes clusters.
+Teleport is a unified access plane for infrastructure:
+
+* [Clusters of Linux servers](https://goteleport.com/teleport/docs/quickstart/) via SSH or SSH-over-HTTPS in a browser
+* [Kubernetes clusters](https://goteleport.com/teleport/docs/kubernetes-ssh/)
+* [Web Applications](https://goteleport.com/teleport/docs/application-access/)
+* [Databases - (Postgres Preview)](https://goteleport.com/teleport/docs/preview/teleport-database-access/)
 
 It is intended to be used instead or together with `sshd` for organizations who
 need:
 
 * SSH audit with session recording/replay.
 * Kubernetes API Access with audit and `kubectl exec` recording/replay.
 * Easily manage trust between teams, organizations and data centers.
-* Have SSH or Kubernetes access to behind-firewall clusters without any open ports.
+* Have Application, SSH or Kubernetes access to behind-firewall clusters without any open ports.
```

Comment:
The "have" here is redundant in the context of the header i.e. "organisations who need <x>".

```suggestion
* Application, SSH or Kubernetes access to behind-firewall clusters without any open ports.
```